setup.py

```diff
@@ -23,6 +23,7 @@
 # Additional dependencies for examples/tutorials and development
 extra_requirements = [
     'black',
+    'coverage',
```

Comment:
Loosely related: separate out `extra_` dev dependencies and dependencies for tutorials/examples?

Reply:
I think it's been decided that this is OK for now, but there's some ongoing thinking.

Reply:
This has been addressed in https://github.com/pykale/pykale/tree/auto-release-and-setup and PR #60

Reply:
Great, thanks. I'll have a look at that PR as time allows.